src/main/kotlin/stringcalculator/DefaultSeparator.kt

```diff
@@ -0,0 +1,17 @@
+package stringcalculator
+
+object DefaultSeparator {
+
+    const val DEFAULT_SEPARATORS = "[,:]"
+    const val CUSTOM_SEPARATOR_REGEX = "//(.)\n(.*)"
```

Comment:
```suggestion
    private val DEFAULT_SEPARATORS = "[,:]".toRegex()
    private val CUSTOM_SEPARATOR_REGEX = "//(.)\n(.*)".toRegex()
```
다른 곳에서 사용하고 있지 않으니 `private`로 선언해주어도 괜찮을 것 같아요 :)